_posts/en/newsletters/2021-09-22-newsletter.md

```diff
@@ -0,0 +1,165 @@
+  time of this writing.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Lightning Address identifiers announced:**
+  André Neves [announced][tla tweet] the [Lightning Address][lightningaddress
+  website] protocol which wraps [LNURL-pay][lnurl pay] flows into familiar
+  [email-like addresses][lightningaddress diagram].
+
+- **ZEBEDEE releases LN wallet browser extension:**
+  ZEBEDEE [announces][zbe blog] Chrome and Firefox extensions which integrate with its
+  [gaming-focused wallet][zebedee wallet].
+
+- **Specter v1.6.0 supports single-key taproot:**
+  Specter's [v1.6.0][specter v1.6.0] release includes support for both regtest
+  and [signet][topic signet] taproot addresses.
+
+- **Impervious releases LN P2P data API:**
+  [Impervious][impervious website]' framework, built on LND, allows developers
+  to [build][impervious api] P2P data streaming applications over the Lightning Network.
+
+- **Fully Noded v0.2.26 released:**
+  [Fully Noded][fully noded website], a Bitcoin and Lightning wallet for macOS/iOS, adds support for
+  [taproot][topic taproot], [BIP86][], and signet.
+
+## Preparing for taproot #14: testing on signet
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/13-signet.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.21.2rc2][bitcoin core 0.21.2] is a release candidate
+  for a maintenance version of Bitcoin Core.  It contains several bug
+  fixes and small improvements.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Eclair #1932][] implements the revised [anchor outputs][topic anchor
+  outputs] protocol specified in [BOLTs #824][], where all presigned
+  HTLC spends use zero fee, so no fees can be stolen.  See [Newsletter
+  #165][news165 bolts 842] for details.
+
+- [LND #5405][] extends the `updatechanpolicy` RPC so that it reports
+  any channels that are currently unusable due to the current policy (or
+  because of other issues, such as the channel funding transaction still
+  being unconfirmed).
+
+- [LND #5304][] enables it to create and validate macaroons with external
+  permissions unknown to LND itself. This change enables tools such as Lightning
```

Comment:
Could link Lightning Terminal to:

/en/newsletters/2020/08/19/#lightning-labs-releases-lightning-terminal
or 
https://terminal.lightning.engineering/#/